browser/profiles/brave_profile_manager.cc

```diff
@@ -97,10 +98,11 @@ void BraveProfileManager::DoFinalInitForServices(Profile* profile,
     return;
   brave_ads::AdsServiceFactory::GetForProfile(profile);
   brave_rewards::RewardsServiceFactory::GetForProfile(profile);
-  brave_wallet::BraveWalletServiceFactory::GetServiceForContext(profile);
 #if BUILDFLAG(ENABLE_IPFS)
+  brave_wallet::BraveWalletAutoPinServiceFactory::GetServiceForContext(profile);
```

Comment:
why is a brave wallet service guarded by enabled_ipfs, they are not guarded by enable_ipfs in the BUILD.gn 

Also why are these being called here in the first place? You should be using `BrowserContextKeyedServiceFactory:: ServiceIsCreatedWithContext`